reporter/src/main/scala/com/typesafe/tools/mima/lib/analyze/method/MethodChecker.scala

```diff
@@ -37,7 +37,9 @@ private[analyze] abstract class BaseMethodChecker extends Checker[MemberInfo, Cl
 
 private[analyze] class ClassMethodChecker extends BaseMethodChecker {
   def check(method: MemberInfo, inclazz: ClassInfo): Option[Problem] = {
-    if (method.isDeferred)
+    if (method.nonBridge && method.nonAccessible)
```

Comment:
It's in the PR description. Erased apply methods have both flags set, and we can't ignore them.